druid/src/win_handler.rs

```diff
@@ -93,7 +93,10 @@ struct Inner<T> {
     windows: Windows<T>,
     /// the application-level menu, only set on macos and only if there
     /// are no open windows.
-    root_menu: Option<MenuDesc<T>>,
+    root_menu: Option<MenuManager<T>>,
+    /// The id of the most-recently-focused window that has a menu. On macOS, this
+    /// is the window that's currently in charge of the app menu.
+    menu_window: Option<WindowId>,
```

Comment:
ah okay I'm seeing an 'unused field' warning on this for non-mac platforms. Maybe worth an 'allow unused' or whatever?

Reply:
Strange that it doesn't complain for me locally... maybe it's a different behavior in nightly? :shrug: Anyway, easy to fix..